webrender_api/src/display_item.rs

```diff
@@ -67,9 +67,6 @@ pub struct PrimitiveInfo<T> {
     pub clip_rect: TypedRect<f32, T>,
     pub is_backface_visible: bool,
     pub tag: Option<ItemTag>,
-    #[cfg(debug_assertions)]
-    #[serde(skip_serializing, default)]
```

Comment:
just for the record, the issue is this line, which shouldn't be included (or should be `(skip_serializing, skip_deserializing)`).